File: Servers/Python/Aiohttp/routers/db_select.py
from aiohttp import web

from Servers.Python.utils.db.sql_query import *
from Servers.Python.utils.path import DbSelect

router = web.RouteTableDef()


@router.get(DbSelect.one_record)
async def one_record(request: web.Request) -> web.Response:
    async with request.app['pool'].acquire() as con:
        return web.json_response({'len': len(
            await con.fetch(
                QUERY_1
            ))})


@router.get(DbSelect.one_k_records)
async def one_k_records(request: web.Request) -> web.Response:
    async with request.app['pool'].acquire() as con:
        return web.json_response({'len': len(
            await con.fetch(
                QUERY_1K
            ))})


@router.get(DbSelect.ten_k_records)
async def ten_k_records(request: web.Request) -> web.Response:
    async with request.app['pool'].acquire() as con:
        return web.json_response({'len': len(
            await con.fetch(
                QUERY_10K
            ))})


@router.get(DbSelect.hundred_k_records)
async def hundred_k_records(request: web.Request) -> web.Response:
    async with request.app['pool'].acquire() as con:
        return web.json_response({'len': len(
            await con.fetch(
                QUERY_100K
            ))})

File: Servers/JS/Express/routers/dbSelect.js
const router = require("express").Router();

const { pool } = require("../db");
const { QUERY_1, QUERY_10K, QUERY_100K, QUERY_1K } = require("../../db/query");
const { dbSelect } = require("../../utils/paths");

router.get(dbSelect.oneRecord, async (_, res) => {
  const { rows } = await pool.query(QUERY_1);
  res.send({ len: rows.length });
});
router.get(dbSelect.oneKRecord, async (_, res) => {
  const { rows } = await pool.query(QUERY_1K);
  res.send({ len: rows.length });
});
router.get(dbSelect.tenKRecords, async (_, res) => {
  const { rows } = await pool.query(QUERY_10K);
  res.send({ len: rows.length });
});
router.get(dbSelect.hundredKRecord, async (_, res) => {
  const { rows } = await pool.query(QUERY_100K);
  res.send({ len: rows.length });
});

module.exports = router;
